### DISCUSSION TITLE
Guide to understand the difference between different models

### DISCUSSION THREAD

**Discussion by damiensellier** [Anjok07/ultimatevocalremovergui#1608] [General]
Hi,
I really want to dive deeper into UVR but I struggle to understand the different algorithm models.
I followed different tutorials I found on youtube or reddit and on this discussion board and got good or so-so results depending on the options. But there's no such thing as a comprehensive guide about the different models and what to use for particular purpose.

Obviously models have been designed with a particular task in mind but if you read : **UVR-MDX-NET Inst HQ3** that doesn't really help. Especially,what's the difference between **HQ3** vs **HQ4**.

A wiki section on this github would be great to provide those details and what user could expect from a certain **process type** and **model**.

I'd be glad to participate in that if I could find the details somewhere.

Thanks

Damien

**Comment by MonolithFoundation** [comment]
Same question, not very clear.

**Comment by xzh2004** [comment]
Same question

**Comment by Liubomyrka** [comment]
+

**Comment by scryptkyddy** [comment]
UVR should have a model wiki or something.

**Comment by IC-Alchemy** [comment]
I'd recommend asking any AI about the different models and settings.

But a few quick tips:

- If a set of models has sequential numbers like HQ1, HQ2, HQ3  You want to use the highest number.  These are version numbers in all cases as far as I can tell.
-  VR models are the oldest models.  Lower quality but run much faster on older computers.
- MDX models score the highest on tests and have the most training.  Many claim MDX models are the best.
- MUCS models are the best for splitting a song into individual parts 

htdemucs_6s = Vocals, Drums, Bass, Guitar, Piano, Other
